rp2040-hal/src/timer.rs

```diff
@@ -3,12 +3,14 @@
 
 use embedded_time::duration::Microseconds;
 
-use crate::pac::{RESETS, TIMER};
+use crate::pac::{timer, RESETS, TIMER};
 use crate::resets::SubsystemReset;
+use core::marker::PhantomData;
 
 /// Timer peripheral
 pub struct Timer {
     timer: TIMER,
+    alarms: [bool; 4],
 }
 
 // Safety: All access is read-only.
```

Comment:
This is no longer true

Reply:
Removed safety comment, this is a breaking change though